--- hyprland.conf	2024-02-19 15:34:53.275782013 +0800
+++ hyprland.conf.dp1	2024-02-19 15:42:19.165800236 +0800
@@ -241,7 +241,7 @@
 bind = $mainMod, print, exec, grim -g "$(slurp -d)" - | wl-copy
 
 bind = CTRL ALT, V, exec, cliphist list | rofi -dmenu | cliphist decode | wl-copy
-bind = CTRL ALT, W, exec, wp="~/Wallpapers/"$(ls ~/Wallpapers/ | sort -u | rofi -dmenu) && hyprctl hyprpaper preload $wp && hyprctl hyprpaper wallpaper "LVDS-1,$wp" && hyprctl hyprpaper unload $wp
+bind = CTRL ALT, W, exec, wp="~/Wallpapers/"$(ls ~/Wallpapers/ | sort -u | rofi -dmenu) && hyprctl hyprpaper preload $wp && hyprctl hyprpaper wallpaper "LVDS-1,$wp" && hyprctl hyprpaper unload $wp
 bind = CTRL ALT, S, exec, notify-send "$(trans "$(tac .trans_history | head -n10 | rofi -dmenu | tee -a .trans_history)" -no-ansi -v)"
 
 bind = CTRL ALT, B, exec, playerctl previous
